package.json

```diff
@@ -2,7 +2,7 @@
   "name": "helloworld-cloud",
   "version": "0.2.0",
   "dependencies": {
-    "body-parser": "~1.0.2",
+    "body-parser": "~1.17.2",
```

Comment:
BTW, if you decide to do it later, I've created an Epic to track dependencies updates as most Cloud apps will need this too. Feel free to create an issue for helloworld-cloud and link it to the Epic
https://issues.jboss.org/browse/RHMAP-16281